app/assets/javascripts/templates/price_breakdown.html.haml

```diff
@@ -2,6 +2,7 @@
   %span.joyride-nub.top
   .background{ng: {click: "tt_isOpen = false"}}
   .joyride-content-wrapper
+    %h6= t("price_breakdown")
```

Comment:
Ideally we would use a specific namespace here under js. so that we continue our slow move into https://github.com/openfoodfoundation/openfoodnetwork/issues/2721

Reply:
Yes. More importantly, I realised that this is a JS template and we need to translate in JS to show other languages than the instance's default. I fixed that in another commit.